demo/src/main/java/com/google/android/exoplayer/demo/Samples.java

```diff
@@ -125,6 +125,8 @@ public Sample(String name, String contentId, String uri, int type) {
         + "prog_index.m3u8", PlayerActivity.TYPE_HLS),
     new Sample("Apple ID3 metadata", "http://devimages.apple.com/samplecode/adDemo/ad.m3u8",
         PlayerActivity.TYPE_HLS),
+    new Sample("Apple WebVtt", "https://devimages.apple.com.edgekey.net/streaming/examples/bipbop_16x9/bipbop_16x9_variant.m3u8",
```

Comment:
This exact example is already added at L117, so you can revert changes to this file.